schemas/webpackOptionsSchema.json

```diff
@@ -750,7 +750,7 @@
           "type": "boolean"
         },
         "maxInitialChunkSize": {
-          "description": "Total size of all initial chunks (in kb)",
+          "description": "Total size of all initial chunks (in bytes)",
           "type": "number"
```

Comment:
Is there precedence in the Webpack community for favoring byte-definitions for configuration over KBs? (IIRC it made getting budgets implemented more straight-forward but just wanted to check) :)